lib/view/widget/RootWidget.dart

```diff
@@ -15,7 +16,7 @@ abstract class RootWidget<T extends RootViewModel> extends StatelessWidget {
         return widget(model);
       },
       viewModelBuilder: () => _model,
-      onModelReady: (model) => model.initialize(),
+      onModelReady: (model) => model.initialize(Get.arguments),
```

Comment:
Jmmm, I don't feel comfortable using `Get.arguments`, what do you thing about moving this to navigator or something?? (I know that is also bad on the current version of the repo)

Reply:
From my POV, retrieving passed arguments is not really a Navigator's responsibility.

An alternative could be using a getArguments() abstract method in RootWidget so the descendants could decide how to provide them. But don't really like it, since there will be a lot of widgets with no arguments which would be forced to override that method:

![image](https://user-images.githubusercontent.com/19324743/108194697-373f1680-7117-11eb-8ade-168a470780d6.png)

![image](https://user-images.githubusercontent.com/19324743/108195070-a1f05200-7117-11eb-88a5-b1e6aee20533.png)